### PR TITLE
added homepage UI

### PR DESCRIPTION
**Pull request by cjinc121**
- created homepage UI
Screenshot (134)

### CODE REVIEW

File: notify/src/component/left-sidebar/leftSidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import "./leftSidebar.css";
+import {AiFillHome} from "react-icons/ai";
+import {MdLabelImportant} from 'react-icons/md';
+import {BiArchiveIn} from "react-icons/bi";
+import {BsTrashFill} from "react-icons/bs";
+import {CgProfile} from "react-icons/cg";
+const LeftSidebar=()=>{
+return <div className="sidebar">
+   <div className="sidebar-heading"><AiFillHome className="sidebar-icon"/>Home</div>
```
Comment (SukanyaSen530) [inline notify/src/component/left-sidebar/leftSidebar.jsx:9]: Can be replaced with Link as these are eventually going to be navigation links.

**Review by SukanyaSen530 — APPROVED** [review]

File: notify/src/component/note-card/noteCard.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.note-card {
+  width: 60%;
+  min-height: 7rem;
+  max-height: max-content;
+  border: 1px black solid;
+  background-color: var(--white-color);
+}
+.note-card-title {
+  display: flex;
+  padding: 0.5rem;
```
Comment (psychoactiv) [inline notify/src/component/note-card/noteCard.css:10]: Suggestion- You can use px values less than 1rem

File: notify/src/component/note-card/noteCard.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+.note-icon {
+  cursor: pointer;
+}
+.text {
+  font-size: 1.375rem;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: 1.75rem;
+  padding-bottom: 12px;
+  padding-top: 16px;
```
Comment (psychoactiv) [inline notify/src/component/note-card/noteCard.css]: And rem for values more than 10px

**Review by psychoactiv — APPROVED** [review]
Code looks clean
File structure is also good
This PR is ready to merge

**Pull request merged**